docs/features.md

```diff
@@ -251,7 +251,10 @@ process execution, display the reason for trigger and the associated context.
 
 ## `gef-remote` debugging ##
 
-It is possible to use `gef` in a remote debugging environment.
+It is possible to use `gef` in a remote debugging environment. Required files
+will be automatically downloaded and cached in a temporary directory (`/tmp/gef` on most
+Unix systems). Remember to manually delete the cache if you change the target file or
```

Comment:
Can't it use the `info file` information or something to at least get the binary name, and not sure an old binary ?